### ISSUE TITLE
It doesn't work

### ISSUE THREAD

**Issue by pathbox** [mattn/longcat#3]
Hi,  I try it, but it print the code in the zsh terminal on Mac, not the picture , I don't know why, could you tell me how to fix it

**Comment by mattn** [comment]
Please use terminal which support DRCS Sixel.

**Comment by cmsax** [comment]
iterm works well, the cat background is white but not dark or transparent.

**Comment by mattn** [comment]
Yes, it it problem of iTerm2. Please file the issue. https://gitlab.com/gnachman/iterm2/issues
